frontend/src/styles/MPbuttons.css

```diff
@@ -0,0 +1,23 @@
+.buttonGroup {
+    width: 100%;
+    text-align: center;
+}
+.mp-button{
```

Comment:
A small suggestion would be to add hover styling on the buttons

Reply:
I added some hover styling!